### ISSUE TITLE
Add support for Windows runners in GitHub Actions

### ISSUE THREAD

**Issue by markcerqueira** [ccrma/chuck#181]
We support macOS and Linux so we should add Windows build and unit tests as well.

**Comment by markcerqueira** [comment]
@spencersalazar, @gewang Can one of you please point me to how one builds the Windows version of ChucK?

**Comment by gewang** [comment]
I've been building in Visual Studio (using the project files in src/visual-studio) but I don't how @spencersalazar builds the actual Windows release.

**Comment by vsvsvsvsvs** [comment]
Yes. I've got some students who I told to install the Windows executable version  1.4.1.0. Is this a new version? They can't turn off the virtual machine or add multiple shreds. Should I tell them to download an earlier version?

**Comment by nshaheed** [comment]
With #209 merged, the unit tests now work with windows, I'm curious what else needs to be done to get this set up?
